packages/di/src/lib/injector.ts

```diff
@@ -1,8 +1,10 @@
-import { InjectableMap } from './injectable-map.js';
 import { LifeCycle } from './lifecycle.js';
 import { InjectionToken, Provider, StaticToken } from './provider.js';
 
-export const INJECTABLE_MAP = new InjectableMap();
+/**
+ * Keeps track of all Injectable services and their Injector
+ */
+export const Injectables = new WeakMap<object, Injector>();
```

Comment:
why pascal case? It's not a class nor a type.
suggestions
- `injectables` 
- `_injectables`
- `privateGlobalInjectables`